components/Header/src/index.scss

```diff
@@ -17,4 +18,24 @@
       height: var(--denhaag-header-md-height);
     }
   }
+
+  .denhaag-header__actions {
+    display: none;
+
+    @media (min-width: 1024px) {
+      display: flex;
+    }
+
+    .denhaag-header__actions-action {
+      margin-inline-start: 24px;
```

Comment:
I think we might want to make this a CSS variable too?

Reply:
Done!